website/blog/2024-02-21-deleting-branches-automatically.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Deleting Branches Automatically
+author: Yousaf Nabi
+authorURL: http://twitter.com/you54f
+tags: [features, tags, branches, deployments, releases]
+keywords: [features, tags, branches, deployments, releases]
+date: 2024-02-21
+draft: false
+hide_table_of_contents: true
+---
+
+:::info tl&semi;dr
+
+Call the delete-branch endpoint via an API or our CLI tools, when a branch is deleted in your CI systems. Most CI systems provide event triggers which can be used to detect branch deletion events.
+
+:::
+
+Back in 2021, the Pact Broker introduced the concepts of Branches and environments, rather than overloading tags, for both purposes. This came with a raft of benefits, described in this post.
+
+We've seen rafts of you switch over to the new state of Pact Nirvana, utilizing branches and environments, to their full effect and today I'd like to share a little tip, to help keep your Pact Broker working at it's optimum.
```

Comment:
As you used rafts in the previous sentence, maybe change "rafts" here with "loads" or "heaps" or something equally exciting